week4-assignment/src/pages/MypagePage.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+import MypageHeader from '../components/Mypage/MypageHeader';
+import MypageHobby from '../components/Mypage/MypageHobby';
+import MypageInfo from '../components/Mypage/MypageInfo';
+
+const MypagePage = () => {
+  const [selectedComponent, setSelectedComponent] = useState<string>('취미');

```

Comment:
```suggestion
  const [selectedComponent, setSelectedComponent] = useState<'취미' | '내 정보'>('취미');
```

이와 같은 선택 메뉴 처럼 특정 값만 오는 state의 경우에는
string타입 보다는 올 수 있는 값들만 명시해주면 조금 더 안전한 코드를 작성할 수 있을 것 같습니다!

Reply:
오.. 👍👍 올 수 있는 값만 명시하면 더 안전한 코드가 되는군요..! 수정했습니다!